### PR TITLE
Mekasuit render fix backport.

### PR DESCRIPTION
**Pull request by KostromDan**
Fixes https://github.com/Asek3/Oculus/issues/170

### CODE REVIEW

**Comment by KostromDan** [comment]
image

File: src/main/java/net/coderbot/iris/pipeline/newshader/ShaderKey.java [hunk 1]
```diff
@@ -58,22 +58,25 @@ public enum ShaderKey {
...
+	SHADOW_LEASH           (ProgramId.Shadow,      		AlphaTests.OFF,             DefaultVertexFormat.POSITION_COLOR_LIGHTMAP,    FogMode.OFF,          LightingModel.LIGHTMAP  ),
+	SHADOW_LIGHTNING       (ProgramId.Shadow,      		AlphaTests.OFF,             DefaultVertexFormat.POSITION_COLOR,     		FogMode.OFF,          LightingModel.FULLBRIGHT),
+	SHADOW_PARTICLES       (ProgramId.Shadow,      		AlphaTests.ONE_TENTH_ALPHA, DefaultVertexFormat.PARTICLE,	     			FogMode.OFF,          LightingModel.LIGHTMAP  ),
+	SHADOW_TEXT           (ProgramId.Shadow,       		AlphaTests.NON_ZERO_ALPHA,  IrisVertexFormats.ENTITY                       ,FogMode.OFF,          LightingModel.LIGHTMAP  ),
+	SHADOW_TEXT_INTENSITY (ProgramId.Shadow,       		AlphaTests.NON_ZERO_ALPHA,  IrisVertexFormats.ENTITY                       ,FogMode.OFF,          LightingModel.LIGHTMAP  ),
+	MEKANISM_FLAME_SHADOW (ProgramId.ShadowEntities,	AlphaTests.ONE_TENTH_ALPHA,	DefaultVertexFormat.POSITION_TEX_COLOR, 		FogMode.OFF, 		  LightingModel.LIGHTMAP  );
+
```
Comment (dima-dencep) [inline src/main/java/net/coderbot/iris/pipeline/newshader/ShaderKey.java]: Revert

**Review by dima-dencep — CHANGES_REQUESTED** [review]

File: src/main/java/net/coderbot/iris/pipeline/newshader/ShaderKey.java (hunk 1 above)
Comment (dima-dencep) [reply src/main/java/net/coderbot/iris/pipeline/newshader/ShaderKey.java]: Oops, I accidentally highlighted an extra line (the last one with `MEKANISM_FLAME_SHADOW`), it doesn't need to be rolled back

**Comment by KostromDan** [comment]
Noticed differences in tab length between GitHub and IDEs. If it looked correct in the IDE, it looked wrong on GitHub. When I fixed it in the GitHub editor and copied it back to the IDE, it started looking incorrect there. So, I replaced tabs with spaces, and now it looks correct in both places.

**Review by dima-dencep — CHANGES_REQUESTED** [review]
I hate to break it to you, but you made even more unnecessary changes

**Comment by dima-dencep** [comment]
> Noticed differences in tab length between GitHub and IDEs

skill issue

**Comment by KostromDan** [comment]
> > Noticed differences in tab length between GitHub and IDEs
> 
> skill issue

https://github.com/Asek3/Oculus/blob/1.19.2/src/main/java/net/coderbot/iris/pipeline/newshader/ShaderKey.java#L63
Wdym, is it correct? In IntelliJ IDEA or VS Code it looks correct, but not on GitHub.
How it looks on GitHub:
image
How it looks in IntelliJ IDEA(tabs are highlited):
image

**Comment by KostromDan** [comment]
If you want I'll revert my fixes and apply only requested, but I think it should look correct on both IDEs and GitHub.

**Comment by dima-dencep** [comment]
I don't care how it looks to you, you can see how it looks to everyone in your own pr and I don't understand why you don't see these changes in diff

**Review by dima-dencep — APPROVED** [review]

**Pull request merged**